big-o/big-o-notation.md

```diff
@@ -76,21 +76,36 @@ These complexities are ordered! We can describe their order in all of these ways
 | O(n)             | Linear      | The algorithm will grow in time or space directly proprotional to the input size. The complexity increases at the same rate that the input increases.                                                                                                                                         |
 | O(n log n)       | Log Linear  | A term used to describe an algorithm which will grow in time or space complexity proportional to the n log n of the input size. "n log n" means that the input size is multiplied by the base-2 log of the input size.                                                                                                                                                              |
 | O(n<sup>2</sup>) | Quadratic   | The algorithm will have a runtime or memory usage proportional to the size of the input squared. This often involves 2 nested loops.                                                                                                                                                    |
-| O(2<sup>n</sup>) | Exponential | The algorithm's complexity doubles each time the input size increases by one.                                                                                                                                                                                                                 |
+| O(2<sup>n</sup>) | Exponential | The algorithm's complexity doubles each time the input size increases by one.                                                                                                                                      |
 
+
+You can see a chart comparing these complexities below.  This chart illustrates for a given input size, how the output of the given expressions change.  Look at the curve of each line.  Which curve accellerates most quickly?  Which curve barely increases at all as the input size increases.
+
+![Complexities comparison chart](images/time-complexities.png)
```

Comment:
Chris hehehehe sorry to be picky about files (actually, you know I'm not sorry, but I'm still sorry)

Could you throw all assets into `../assets` (aka there's an `assets` folder on the root level)? Alooosososososo my preferred fiiile naming pattern for this would be `big-o-notation_time-complexities.png` loooool

Also, weirdly enough, this picture doesn't show up in my GH preview anyway. Is that a problem on my preview end?

Reply:
I've thrown it into assets and renamed it.

I think the preview must be on your end as it shows up for me and in the branch on github.